### ISSUE TITLE
Cannot install v1.4.1 on windows, 1.4.0 installs fine

### ISSUE THREAD

**Issue by nickjillings** [supermihi/pytaglib#37]
Error when running 'pip3 install pytaglib' on Windows 10. It seems a file is missing, however installing v1.4.0 is fine.

The output from the log:
```
Collecting pytaglib
  Downloading pytaglib-1.4.1.tar.gz (108kB)
    100% |################################| 112kB 479kB/s
Building wheels for collected packages: pytaglib
  Running setup.py bdist_wheel for pytaglib ... error
  Complete output from command [redacted]\python\python35\python.exe -u -c "import setuptools, tokenize;__file__='[redacted]\\Temp\\pip-build-m29m2408\\pytaglib\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d [redacted]\Temp\tmpqb352i9fpip-wheel- --python-tag cp35:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.5
  copying src\pyprinttags.py -> build\lib.win-amd64-3.5
  running build_ext
  building 'taglib' extension
  creating build\temp.win-amd64-3.5
  creating build\temp.win-amd64-3.5\Release
  creating build\temp.win-amd64-3.5\Release\src
  C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -DTAGLIB_STATIC -IC:\Libraries\taglib\include -Ic:\users\nicholas\appdata\local\programs\python\python35\include -I[redacted]\python\python35\include "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\INCLUDE" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-IC:\Program Files (x86)\Windows Kits\8.1\include\\shared" "-IC:\Program Files (x86)\Windows Kits\8.1\include\\um" "-IC:\Program Files (x86)\Windows Kits\8.1\include\\winrt" /EHsc /Tpsrc\taglib.cpp /Fobuild\temp.win-amd64-3.5\Release\src\taglib.obj
  taglib.cpp
  src\taglib.cpp(469): fatal error C1083: Cannot open include file: 'taglib/tstring.h': No such file or directory
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2

  ----------------------------------------
  Failed building wheel for pytaglib
```

**Comment by supermihi** [comment]
Hi, thanks for your report.

The reason was that there was a precompiled binary package available for v1.4.0 and Python 3.5, but for v1.4.1 I had only uploaded a binary package for Python 3.6.

Now, I have just uploaded Windows binary packages for Python 3.5 *and* 3.6, each for both x32 and x64. If you try again, the installation should work.

**Comment by supermihi** [comment]
I am considering this issue fixed.
